nixos/tests/all-tests.nix

```diff
@@ -320,6 +320,7 @@ in {
   mimir = handleTest ./mimir.nix {};
   garage = handleTest ./garage {};
   gemstash = handleTest ./gemstash.nix {};
+  geoserver = runTest ./geoserver.nix;
```

Comment:
@sikmir, I am not sure if I should use handleTest or runTest here. Do you have some preference ?